app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -92,10 +91,17 @@ def update
       if params[:product_ids].nil?
         @products = @alternate_content_source.products
       elsif params[:product_ids] == []
+        # Check that this acs is simplified before allowing products to be cleared
```

Comment:
Hopefully this doesn't complicated things too much, but we should also disallow users from changing the alternate content source type or the content type on update. It would require recreating all of the backend Pulp entities and the base url + subpaths would likely no longer make sense.

Reply:
Features added :)